### ISSUE TITLE
Foxpro hangs after shell command

### ISSUE THREAD

**Issue by severach** [dosemu2/fdpp#228]
Two commands `!dir` `quit` and Foxpro hangs at 100% CPU. No problems with FreeDOS. I don't know when it started as I've been using FreeDOS for a while.

88c1e6f still does it
af5afd3 won't run any more

**Comment by stsp** [comment]
Please attach foxpro.

**Comment by severach** [comment]
Only two commands are needed. Plenty of commands after `!dir` hang but these are the simplest.

````
!dir
quit
````

**Comment by stsp** [comment]
> Please attach foxpro.

Please concentrate and do what was
said, or there is no talk.

**Comment by severach** [comment]
Foxpro 2.6a DOS executable is copyright program. Can't attach that here. Other than test program and executable I don't know what other "Foxpro" you want.

Foxpro executable seems to be available at WinWorld and other sites.

**Comment by stsp** [comment]
Seems to be working well with fdpp+freecom.
So perhaps a comcom32 bug.

**Comment by severach** [comment]
Yes, using comcom32. comcom32 same version for a long time so if it is, it's something in fdpp that broke comcom32.

````
$ pacman -Q comcom32 fdpp dosemu
comcom32 0.0.alpha_3-1
fdpp-git 1.6.r171.gbd42012-1
dosemu2-git 2.0pre9.dev.r1373.gc0344cba6-1
````

I only use freedos+freecom and fdpp+comcom32. Should I offer fdpp+freecom?
